crux-llvm/test-data/golden/shrink.c.orig

```diff
@@ -0,0 +1,27 @@
+unsigned int addflt(unsigned int a , unsigned int b )
```

Comment:
There is already a test case for this [here](https://github.com/GaloisInc/crucible/blob/796d1f595e7cbcd421e68050fa5cc864327b7aab/crux-llvm/test-data/golden/T778.c). Perhaps they should be consolidated?

Reply:
Maybe... but I want to use the `.ll` here to really be sure that we retain the problematic def/use inversion across future compiler changes. What do you think?